content/docs/12.5-vertex/00-fips/01-fips-compliant-components.md

```diff
@@ -46,25 +46,128 @@ Palette VerteX provides FIPS-compliant infrastructure components in Kubernetes c
+| Scheduler | The scheduler is a daemon that finds the best node for a pod, based on the scheduling requirements you specify. |
+| Kubelet | The kubelet is the primary *node agent* that is deployed on each node. |
+| Kube-proxy | The kube-proxy is a network proxy that runs on each node in your cluster, implementing part of the Kubernetes Service concept. |
+| Kubeadm | Kubeadm is a tool built to provide best-practice “fast paths” for creating Kubernetes clusters. |
+| Kubectl | Kubectl is a command line interface for issuing commands against Kubernetes clusters. |
+
+
+### Auxiliary Kubernetes Components
 
-## Kubernetes
+| **Component** | **Description** |
+| --- | --- |
+| CoreDNS | CoreDNS is a DNS server that is deployed as a cluster DNS service. |
+| Etcd | Etcd is a distributed key-value store that is used as Kubernetes’ backing store for all cluster data. |
+| Metrics Server | Metrics Server is a scalable, efficient source of container resource metrics for Kubernetes built-in autoscaling pipelines. |
+| Ingress Controller| Nginx is used as the ingress controller. An ingress controller is a piece of software that provides reverse proxy, configurable traffic routing, and TLS termination for Kubernetes services. |
+| Nginx Server| The Nginx server is a web server that can also be used as a reverse proxy, load balancer, mail proxy, and HTTP cache. |
+| Nginx Ingress Controller| The Nginx ingress controller is an ingress controller that uses ConfigMap to store the Nginx configuration. |
 
-Refer to the [Palette eXtended Kubernetes (PXK)](/integrations/kubernetes) and [Palette eXtended Kubernetes-Edge (PXK-E)](/integrations/kubernetes-edge)  to learn about these security-hardened packs.
 
-<!-- <br />
+### Runtime Components
 
-- Helm
-- Open Container Initiative (OCI) Registry As Storage (ORAS)
-- DevSpace open-source developer tool for Kubernetes
-- Kubectl command line tool
-- Kustomize to enable customizing YAML files
-- Amazon Web Services (AWS) IAM Authenticator
-- etcd -->
+| **Component** | **Description** |
+| --- | --- |
+| containerd |  Containerd is an industry-standard container runtime with an emphasis on simplicity, robustness, and portability. |
+| containerd-shim | Containerd-shim is a shim used by containerd to launch containers. |
+| containerd-shim-runc-v1 | Containerd-shim-runc-v1 is a shim used by containerd to launch containers. |
+| containerd-shim-runc-v2 | Containerd-shim-runc-v2 is a shim used by containerd to launch containers. |
+| ctr| Ctr is a command line interface for containerd. |
+| crictl | Crictl is a command line interface for CRI-compatible container runtimes. |
+| runc | Runc is a CLI tool for spawning and running containers according to the OCI specification. |
 
 
+### Container Network Interface Components
+
+| **Component** | **Description** |
+| --- | --- |
+| Calico | Calico is a CNI plugin that provides networking and network policy for Kubernetes clusters. |
```

Comment:
Do we spell out CNI here, or not because it's in the heading? Or do we add (CNI) in the heading?

Same for CSI in next section.